rpc/erigon.go

```diff
@@ -283,6 +283,11 @@ func (client *ErigonClient) GetBlock(number int64, traceMode string) (*types.Eth
 					}
 
 					c.Transactions[trace.TransactionPosition].Itx = append(c.Transactions[trace.TransactionPosition].Itx, tracePb)
+					if len(trace.TraceAddress) == 0 {
```

Comment:
Rare comment by me on code: I don't know if we want to use Erigon specific API. If we ever consider changing the EL this might break stuff. If I interpret this incorrectly, just ignore me :)

Reply:
That's not touched, just using another field of the reply. We are already using an Erigon specific endpoint [here](https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2618/files#diff-a36de22b09d32f97df1d4b745a6cc61e3242d7693494334a65dec6747032932aL228), but there's a fallback to geth in place [here](https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2618/files#diff-a36de22b09d32f97df1d4b745a6cc61e3242d7693494334a65dec6747032932aL290). Both cases should have the same result.